api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/core/logout/LogoutProperties.java

```diff
@@ -25,7 +26,7 @@
      * is indicated and extracted by a parameter name of your choosing here. If none specified,
      * the default will be used as {@code service}.
      */
-    private String redirectParameter;
+    private String redirectParameter = CasProtocolConstants.PARAMETER_SERVICE;
```

Comment:
Please revert this change; it's preferable to not depend on `CasProtocolConstants` if possible.

Reply:
Hi,

So, should I just default it like this ?

```
private String redirectParameter = "service"
```

If it is not defined, the second bug described in this PR will occurs. Or should I default it in the OidcLogoutEndpointController.java file ?

Regards,
Julien

Reply:
The second option, where you default it much like the LogoutAction, as:

```
StringUtils.defaultIfEmpty(logoutProperties.getRedirectParameter(), CasProtocolConstants.PARAMETER_SERVICE);
```

Reply:
On second thought using `"service"` as the default is perfectly fine too :)

Reply:
Ok.

Will do it in a few hours. :)

Reply:
Done.